html/changelogs/DeityLink_4809.yml

```diff
@@ -0,0 +1,6 @@
+author: Deity Link
+delete-after: true
+changes:
+  - bugfix: Admins now get notified when one of them gives an achievement. (Including who gave it)
+  - bugfix: Ghosts now get notified when someone receives an achievement.
```

Comment:
these are kinda missing features that should have been there when I first added achievement.